svf/include/SVFIR/SVFVariables.h

```diff
@@ -582,7 +555,7 @@ class BaseObjVar : public ObjVar
 
 protected:
     /// Constructor to create empty ObjVar (for SVFIRReader/deserialization)
-    BaseObjVar(NodeID i, PNODEK ty = BaseObjNode) : ObjVar(i, ty) {}
+    BaseObjVar(NodeID i, PNODEK ty = BaseObjNode, const ICFGNode* node = nullptr) : ObjVar(i, ty), icfgNode(node) {}
 
```

Comment:
can icfgNode be a nullptr? It would be better to pass nullptr as an argument if it really has a null value.